permissions/src/iosMain/kotlin/storage/StoragePermissionManager.kt

```diff
@@ -0,0 +1,85 @@
+/*
+ Copyright (c) 2020. Splendo Consulting B.V. The Netherlands
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+ */
+
+package com.splendo.kaluga.permissions.storage
+
+import com.splendo.kaluga.base.mainContinuation
+import com.splendo.kaluga.base.runBlocking
+import com.splendo.kaluga.permissions.*
+import platform.Foundation.NSBundle
+import platform.Photos.*
+
+actual class StoragePermissionManager(
```

Comment:
Not a good name, there is no storage permissions concept at iOS.
Maybe make it clear: Photos or PhotoLibrary permissions manager?

Reply:
Well I dislike photos because on Android its just SD card access. Not necessarily photos. But they are on a similar level imo. So I went with the (broader) Android perspective for the naming convension.

Reply:
Then we have to split this permission to platforms. Leave Storage for Android and Photos on iOS. Now this Storage on iOS side looks very very weird

Reply:
Yes. For me would be confusing as well, but the permission manager will be used mostly from shared so would leave this name, but add few more type aliases for iOS developers

Reply:
Or probably change 'Storage' to 'PhoneSystemStorage' or 'SystemFileStorage' or 'UniversalStorage'